autoscheduler/frontend/src/redux/reducers/courseCards.ts

```diff
@@ -61,18 +121,25 @@ function getStateAfterExpanding(
   courseCardUpdates: CourseCardOptions,
 ): CourseCardArray {
   // Determine new number of cards created
-  const numCardsCreated = Math.max(state.numCardsCreated, indexToExpand + 1);
+  const numCardsCreated = Math.max(Number.isNaN(state.numCardsCreated) ? 0 : state.numCardsCreated,
```

Comment:
What's the need for this `Number.isNaN` addition?

Reply:
On page reload I believe there's a second while the fetch for session data is processing that has undefined for numCardsCreated. This function runs there and ends up crashing the whole website without it.